Content.Server/Flash/FlashSystem.cs

```diff
@@ -155,6 +155,10 @@ public void FlashArea(EntityUid source, EntityUid? user, float range, float dura
 
             foreach (var entity in _entityLookup.GetEntitiesInRange(transform.Coordinates, range))
             {
+                //cut to a circle. for the aesthetics of cleansing Shadow Kudzu. The square-shaped removal from the flash looked ugly
```

Comment:
this should be in the shadow event handler not here

Reply:
shadow kudzu should not be in flash system, this should be in its flash attempt event handler not thrown here

Reply:
I can delete the comment. It's basically a logical addition, since light propagation in the real world doesn't work square.